components/WorkSectionAsGallery.js

```diff
@@ -0,0 +1,221 @@
+import React, { PureComponent } from 'react';
+
+import Link from 'next/link';
+
+import PropTypes from 'prop-types';
+
+import { ContentfulMedia, SimpleFragment } from 'models';
+
+import { Markdown, Image } from 'components/base';
+
+import get from 'utils/get';
+import flattenImageData from 'utils/flattenImageData';
+
+const pauseStartVideo = (videoId) => {
+  const video = document.getElementById(videoId);
```

Comment:
i wouldn't trust that the videos won't be duplicated, human errors happen all the time (esp if this were a client project). we should actually use [refs](https://reactjs.org/docs/refs-and-the-dom.html) to target the current video for media playback